File: level1/ex10.js
function solution(participant, completion) {
  let answer = '';

  participant.sort();
  completion.sort();

  const n = participant.length;

  for (let i = 0; i < n; i++) {
    if (completion[i] !== participant[i]) {
      answer = participant[i];
      break;
    }
  }

  return answer;
}

File: level1/ex10.py
def solution(participant, completion):
    # 참가자 오름차순 정렬
    participant = sorted(participant)
    # 완주 선수 오름차순 정렬
    completion = sorted(completion)
    
    for i in range(len(completion)):
        if participant[i] != completion[i]:
            return participant[i]
        
    return participant[-1]